src/components/RichText.ts

```diff
@@ -20,27 +20,35 @@ export interface CommonRichTextProps {
     customOptions: { option: string }[];
     minNumberOfLines: number;
     maxNumberOfLines: number;
+    recreate?: boolean;
 }
 
 export interface RichTextProps extends CommonRichTextProps {
     className?: string;
-    onChange?: (value: string) => void;
+    onChange?: (value?: string) => void;
+    onBlur?: () => void;
+    update: boolean;
+    getQuill?: (quill: Quill.Quill) => void;
```

Comment:
Function is still reference but is not used actively in the container

Reply:
Removed.